### DISCUSSION TITLE
There is a problem when using the tools

### DISCUSSION THREAD

**Discussion by gatoencerradoensucasa** [new92/instatools#25] [Q&A]
![Screenshot_2024-01-05_12_35_18](https://github.com/new92/InstaTools/assets/155764156/35eac088-46c4-473e-bdca-42febd646807)
When a try to use every tool, this img was researcher, I writes yes to the consent and this always happens, could anyone help me please

**Comment by new92** [comment]
@gatoencerradoensucasa try using instatools with root privileges.

**Comment by new92** [comment]
@gatoencerradoensucasa Did you manage to solve the issue ?

**Comment by gatoencerradoensucasa** [comment]
Hey, thanks for the response. It worked in some tools and in other not.
For example this Poirot works (at the end there is an error but dunno why)
![image](https://github.com/new92/InstaTools/assets/155764156/5eca0942-292e-4027-b668-07084ca7ac35)

This is Isver and it gives me this:
![image](https://github.com/new92/InstaTools/assets/155764156/d2fbf0a0-b6d2-4a17-b673-3c4dbc094c96)

I think at first it was the root but I don't know the other ones.
I haven't tried the login/password tools yet but if there is some issue I'll ask

**Comment by new92** [comment]
Hey ! Ok these are clearly bugs of the scripts. Thanks for bringing them to my attention, I'll fix them and I'll contact you again.
